docs/CHANGELOG.rst

```diff
@@ -29,6 +29,7 @@ Fixed
 - Report correct total mapped reads and mapped reads percentage in
   prepeak QC report for ``data:alignment:bam:bowtie2`` inputs in
   ``macs2-callpeak`` process
+- Fix ``prepare-geo-chipseq`` so it collects the right background data
```

Comment:
Do not add additional change log entry, as you can modify (if at all necessary) the existing entry in the unreleased section.